### PR TITLE
cleanup: add missing \n in Jmsg calls where missing

### PR DESCRIPTION
**Pull request by pstorz**
#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Source code changes are understandable
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: core/src/stored/label.cc [hunk 1]
```diff
@@ -128,7 +128,7 @@ int ReadDevVolumeLabel(DeviceControlRecord* dcr)
            _("Wrong Volume mounted on device %s: Wanted %s have %s\n"),
            dev->print_name(), VolName, dev->VolHdr.VolumeName);
       if (!dev->poll && jcr->impl->label_errors++ > 100) {
-        Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s"), jcr->errmsg);
+        Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s\n"), jcr->errmsg);
```
Comment (sduehr) [inline core/src/stored/label.cc:131]:
```suggestion
        Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s"), jcr->errmsg);
```
`jcr->errmsg` already contains `\n`

File: core/src/stored/label.cc [hunk 2]
```diff
@@ -208,7 +208,7 @@ int ReadDevVolumeLabel(DeviceControlRecord* dcr)
          dev->print_name(), dev->VolHdr.LabelType);
     Dmsg1(130, "%s", jcr->errmsg);
     if (!dev->poll && jcr->impl->label_errors++ > 100) {
-      Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s"), jcr->errmsg);
+      Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s\n"), jcr->errmsg);
```
Comment (sduehr) [inline core/src/stored/label.cc:211]:
```suggestion
      Jmsg(jcr, M_FATAL, 0, _("Too many tries: %s"), jcr->errmsg);
```
same here

File: core/src/stored/label.cc [hunk 3]
```diff
@@ -231,7 +231,7 @@ int ReadDevVolumeLabel(DeviceControlRecord* dcr)
      *  => we are in a loop
      */
     if (!dev->poll && jcr->impl->label_errors++ > 100) {
-      Jmsg(jcr, M_FATAL, 0, "Too many tries: %s", jcr->errmsg);
+      Jmsg(jcr, M_FATAL, 0, "Too many tries: %s\n", jcr->errmsg);
```
Comment (sduehr) [inline core/src/stored/label.cc:234]:
```suggestion
      Jmsg(jcr, M_FATAL, 0, "Too many tries: %s", jcr->errmsg);
```
same here

File: core/src/stored/mount.cc [hunk 4]
```diff
@@ -491,7 +491,7 @@ int DeviceControlRecord::CheckVolumeLabel(bool& ask, bool& autochanger)
         Jmsg(jcr, M_WARNING, 0,
              _("Director wanted Volume \"%s\".\n"
                "    Current Volume \"%s\" not acceptable because:\n"
-               "    %s"),
+               "    %s\n"),
```
Comment (sduehr) [inline core/src/stored/mount.cc:494]:
```suggestion
               "    %s"),
```
`CatalogRequest()` already add `\n` to such messages

**Review by sduehr — CHANGES_REQUESTED** [review]
Some of the added `\n` are not needed.

**Review by pstorz — COMMENTED** [review]
Thank you for reviewing!

**Review by sduehr — APPROVED** [review]
Thanks, looks good now.

**Pull request merged**